SimPEG/electromagnetics/static/utils/static_utils.py

```diff
@@ -1859,31 +1343,6 @@ def r_unit(p1, p2):
     return vec, r
 
 
-def getSrc_locs(survey):
```

Comment:
probably just deprecate this function and tell the user to extract from the survey as `locations_a` and `locations_b`

Reply:
actually point the user to `survey.source_locations` in the deprecated version of this function